Formula/payara.rb

```diff
@@ -70,8 +72,7 @@ def plist
   test do
     ENV["GLASSFISH_HOME"] = opt_libexec/"glassfish"
     output = shell_output("#{bin}/asadmin list-domains")
-    assert_equal "domain1 not running", output
-    assert_equal "production not running", output
```

Comment:
Test failed due to `production not running` missing from output.
Not sure if output was changed from 5.192 (May 2019) to 5.2021.4 (May 2021)